[2025-10-14 13:00:42,335 - ERROR] - Traceback (most recent call last):
[2025-10-14 13:00:42,335 - ERROR] -   File "/app/src/TGeniusAI.py", line 7499, in <module>
[2025-10-14 13:00:42,337 - ERROR] - window = VideoAudioManager()
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] - ^
[2025-10-14 13:00:42,338 - ERROR] -   File "/app/src/TGeniusAI.py", line 660, in __init__
[2025-10-14 13:00:42,338 - ERROR] - self.initUI()
[2025-10-14 13:00:42,339 - ERROR] -   File "/app/src/TGeniusAI.py", line 858, in initUI
[2025-10-14 13:00:42,339 - ERROR] - self.recordingDock = self.createRecordingDock()
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,339 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] - ^
[2025-10-14 13:00:42,340 - ERROR] -   File "/app/src/TGeniusAI.py", line 3883, in createRecordingDock
[2025-10-14 13:00:42,341 - ERROR] - monitors = get_monitors()
[2025-10-14 13:00:42,341 - ERROR] - ^
[2025-10-14 13:00:42,341 - ERROR] - ^
[2025-10-14 13:00:42,341 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] - ^
[2025-10-14 13:00:42,342 - ERROR] -   File "/home/jules/.pyenv/versions/3.12.11/lib/python3.12/site-packages/screeninfo/screeninfo.py", line 32, in get_monitors
[2025-10-14 13:00:42,342 - ERROR] - raise ScreenInfoError("No enumerators available")
[2025-10-14 13:00:42,342 - ERROR] - screeninfo.common
[2025-10-14 13:00:42,342 - ERROR] - .
[2025-10-14 13:00:42,342 - ERROR] - ScreenInfoError
[2025-10-14 13:00:42,342 - ERROR] - :
[2025-10-14 13:00:42,342 - ERROR] - No enumerators available
